.github/workflows/ci.yaml

```diff
@@ -215,6 +215,11 @@ jobs:
         working-directory: ./test/project_erlang_windows
         if: ${{ runner.os == 'Windows' && matrix.run-integration-tests }}
 
+      - name: test/project_erlang export package-information
+        run: |
+          gleam export package-information --out="gleam.json"
```

Comment:
Need to `cd` into the package directory here!

Reply:
Fixed in fixup commit